src/main/java/org/zalando/nakadi/config/NakadiSettings.java

```diff
@@ -39,7 +41,11 @@ public NakadiSettings(@Value("${nakadi.topic.max.partitionNum}") final int maxTo
                           @Value("${nakadi.admin.default.dataType}") final String defaultAdminDataType,
                           @Value("${nakadi.admin.default.value}") final String defaultAdminValue,
                           @Value("${nakadi.authz.warnAllDataAccessMessage}") final String warnAllDataAccessMessage,
-                          @Value("${nakadi.topic.compacted.warnMessage}") final String logCompactionWarnMessage) {
+                          @Value("${nakadi.topic.compacted.warnMessage}") final String logCompactionWarnMessage,
+                          @Value("${nakadi.eventType.deletableSubscription.owningApplication}")
```

Comment:
I do not know if you thought about it maybe it should be the list of applications, but I do not insist on that